packages/block-library/src/navigation-menu-item/index.js

```diff
@@ -24,6 +24,10 @@ export const settings = {
 
 	__experimentalDisplayName: 'label',
 
+	supports: {
+		proxyToolbarToParent: true,
```

Comment:
I feel this could be a property of `InnerBlocks` instead.

Reply:
Yes i was thinking about this last night. I can see how it might work. I will try on Wednesday.

Reply:
Done!